bcl/src/bc_spi.c

```diff
@@ -22,14 +22,17 @@ static const uint32_t _bc_spi_mode_table[4] =
     [BC_SPI_MODE_3] = 0x03  // SPI mode of operation is 3 (CPOL = 1, CPHA = 1)
 };
 
-static bc_spi_mode_t _bc_spi_mode;
-static bc_spi_speed_t _bc_spi_speed;
-static void (*_event_handler)(bc_spi_event_t event, void *_event_param);
-static void *_event_param;
-static bool _in_progress;
-static bc_spi_event_t _pending_event;
-static bool _initilized;
-static bc_scheduler_task_id_t _task_id;
+static struct
```

Comment:
All members here should be without underscore becase _bc_spi already represents private variable. Please change. Thank you!